### PR TITLE
jetty 9.4.x make tests working on environment without docker and port 8080 already used

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: jetty-cdi/src/test/java/org/eclipse/jetty/embedded/EmbeddedWeldTest.java [hunk 1]
```diff
@@ -167,7 +167,7 @@ public void testServletContext(String mode) throws Exception
     @Test
     public void testWebappContext() throws Exception
     {
-        Server server = new Server(8080);
+        Server server = new Server(0);
```
Comment (janbartel) [inline jetty-cdi/src/test/java/org/eclipse/jetty/embedded/EmbeddedWeldTest.java:170]: The PR is about infinispan, but this is a change to a Weld cdi test?

File: jetty-cdi/src/test/java/org/eclipse/jetty/embedded/EmbeddedWeldTest.java [hunk 2]
```diff
@@ -199,7 +199,7 @@ public void testWebappContext() throws Exception
     @Test
     public void testWebappContextDiscovered() throws Exception
     {
-        Server server = new Server(8080);
+        Server server = new Server(0);
```
Comment (janbartel) [inline jetty-cdi/src/test/java/org/eclipse/jetty/embedded/EmbeddedWeldTest.java:202]: See previous comment.

**Review by janbartel — CHANGES_REQUESTED** [review]
If you remove the changes to the Weld CDI test and then this should be ok.

File: jetty-cdi/src/test/java/org/eclipse/jetty/embedded/EmbeddedWeldTest.java (hunk 1 above)
Comment (olamy) [reply jetty-cdi/src/test/java/org/eclipse/jetty/embedded/EmbeddedWeldTest.java:170]: I have updated the PR title to explain the goal of this PR. 
Please note each change (testing if docker available and one for the hardcoded port) are already different commits.
So having 2 PRs will have the same result at the end with 2 different commits in the git history and each will have a correct description of the change.

**Review by janbartel — APPROVED** [review]

**Pull request merged**